node/src/command-options/LPosOptions.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+export class LPosOptions implements LPosOptions {
+    public static RANK_REDIS_API = "RANK";
+    public static COUNT_REDIS_API = "COUNT";
+    public static MAXLEN_REDIS_API = "MAXLEN";
+    private rank?: number;
```

Comment:
Is there a subtype for integer numbers?

Reply:
Unfortunately not unless we create a custom integer type. 

- https://www.typescriptlang.org/docs/handbook/basic-types.html
- https://www.tutorialspoint.com/typescript/typescript_types.htm#:~:text=Advertisement-,Built%2Din%20types,-The%20following%20table